crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -733,6 +802,7 @@ public BulkResponse bulkRequest(BulkRequest requests) {
             Error.pop();
             return response;
         } finally {
```

Comment:
I guess since all individual bulk requests (find/update/delete/save) are tracking their own errors, there is no tracking present at the overall bulkrequest level. Also BulkResponse object does not have ability to add errors, so this was left as is. Tracking errors at this level will require changing BulkResponse composition.